terraform/modules/hub/variables.tf

```diff
@@ -13,3 +13,7 @@ variable "tools_account_id" {
 variable "number_of_availability_zones" {
   default = 2
 }
+
+variable "publically_accessible_from_cidrs" {
+  default = ["0.0.0.0/0"]
```

Comment:
are we planning to restrict some ip ranges from public access? if this is to restrict nonpublic I belive we should allow access via vpn only instead of ip restrictions.

Reply:
This is pretty much what this commit prepares for: If we need to, we will allow access only from the office IPs and our VPN.

Reply:
it is already ip restricted as per the branch in verify-infrastructure-config

this is only while we do testing, there is no reason why the public facing load balancer should have ip restrictions